### ISSUE TITLE
proposing changes to files starting with SS*

### ISSUE THREAD

**Issue by vinstah** [silverstripe/silverstripe-framework#9727]
Hi,

There are only a few Files left in there are preceded with SS e.g SS_List, SSViewer, SSTemplateParser

can we change the names in V5?

**Comment by sminnee** [comment]
The benefit isn't zero, but I'm not sure the benefit is greater than the cost.

Assuming that our upgrader code is used to do the rewrite the cost is minimal, although that's perhaps an oversimplification, and in any case I'd rather park this kind of thing until 5 had a clearer arrival date.

**Comment by robbieaverill** [comment]
I agree with @sminnee, but I'm also a strong proponent of eventually achieving PSR-12 compliance, and we won't get there with underscores in our class names.

**Comment by maxime-rainville** [comment]
Didn't make it to CMS5.

**Comment by vinstah** [comment]
how about 5.0.1?

**Comment by GuySartorelli** [comment]
This can't be done outside of a major release.
